sbin/build.sh

```diff
@@ -237,9 +237,15 @@ configureVersionStringParameter() {
   fi
 
   addConfigureArg "--with-vendor-name=" "${BUILD_CONFIG[VENDOR]:-"AdoptOpenJDK"}"
+<<<<<<< HEAD
```

Comment:
looks like your rebase went wrong here

Reply:
@gdams @karianna I think I will have to try asking for help with the rebase